lib/internal/quic/binding.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const {
+  initializeCallbacks,
+} = internalBinding('quic');
+
+// If the initializeCallbacks is undefined, the Node.js binary
```

Comment:
This is probably more accurate:
```suggestion
// If the initializeCallbacks is undefined, the `node` binary
```